packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuthProvision.template.v2.bicep

```diff
@@ -1,4 +1,5 @@
 param provisionParameters object
+@secure()
```

Comment:
@secure() is added above the provisionParameters

Reply:
sure